analyzers/VMRay/vmray.py

```diff
@@ -7,98 +7,230 @@
 class VMRayAnalyzer(Analyzer):
     """
     VMRay analyzer that uses VMRayClient to connect to an VMRay instance. Allows uploading a sample and getting
-    information via hash. More info regarding configuration in the complete documentation.
+    information bac via submission data. More info regarding configuration in the complete documentation.
     """
+
+    _namespace = "VMRay"
+
+    _severity_mapping = {
+        "whitelisted": "safe",
+        "suspicious": "suspicious",
+        "malicious": "malicious",
+        "blacklisted": "malicious",
+    }
+
+    _ioc_mapping = {
+        "domains": "domain",
+        "email_addresses": "email",
+        "emails": "sender",
+        "files": "filename",
+        "ips": "ip_address",
+        "mutexes": "mutex_name",
```

Comment:
Cortex does not know about `mutex_name`. So I guess `other` would be the right data type.